### DISCUSSION TITLE
Visual Studio requirement

### DISCUSSION THREAD

**Discussion by fmiskuly-eastman** [microsoft/react-native-windows#10135] [Q&A] [answered]
In the React native windows tutorial there is a section about running with/without Visual Studio. From my understanding visual studio is a requirement for react native windows. It is not clear to a first time react developer through this verbiage that Visual Studio is a requirement for reasct native windows to run. Additionally, must VS be opened every time while trying to run?

**Comment by chrisglein** [comment, marked as answer]
In the [documentation page on requirements](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) it's not called out explicitly if you follow the recommend path of running the dependency script, but it is there in the manual steps.

No, VS doesn't need to be opened every time when trying to run. See [Running a React Native Windows app](https://microsoft.github.io/react-native-windows/docs/getting-started#running-a-react-native-windows-app)
